code/modules/antagonists/changeling/changeling.dm

```diff
@@ -585,3 +586,7 @@
 
 /datum/antagonist/changeling/xenobio/antag_listing_name()
 	return ..() + "(Xenobio)"
+
+/datum/antagonist/changeling/check_mastery()
+	if(!purchasedanypowers)
+		owner.current.client?.give_award(/datum/award/achievement/antagmastery/changeling, owner.current)
```

Comment:
You master the changeling by not playing changeling and using none of their abilities?